app/src/main/java/com/futurice/freesound/feature/home/HomeFragment.java

```diff
@@ -0,0 +1,158 @@
+    private final Binder binder = new Binder() {
+
+        @Override
+        public void bind(@NonNull final CompositeDisposable d) {
+            Preconditions.checkNotNull(homeFragmentViewModel);
+
+            d.add(homeFragmentViewModel.getImage()
+                                       .subscribeOn(Schedulers.computation())
+                                       .observeOn(mainThread())
+                                       .subscribe(it -> get(picasso).load(it)
+                                                                    .into(avatarImage),
+                                                  e -> Timber.e(e, "Error setting image")));
+
+            d.add(homeFragmentViewModel.getUserName()
+                                       .subscribeOn(Schedulers.computation())
+                                       .observeOn(mainThread())
+                                       .subscribe(it -> get(userName).setText(it),
+                                                  e -> Timber.e(e, "Error setting user")));
+
+            d.add(homeFragmentViewModel.getAbout()
+                                       .subscribeOn(Schedulers.computation())
+                                       .observeOn(mainThread())
+                                       .subscribe(it -> get(about).setText(it),
+                                                  e -> Timber.e(e, "Error setting about")));
+        }
+
+        @Override
+        public void unbind() {
+            // Nothing to do here
+        }
+
+    };
+
+    @NonNull
+    static HomeFragment create() {
+        return new HomeFragment();
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_home, container, false);
+    }
+
+    @Override
+    public void onViewCreated(final View view, @Nullable final Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        unbinder.setIfNone(bind(this, view));
+    }
+
+    @Override
+    public void onDestroyView() {
+        unbinder.getAndClear()
+                .ifSome(Unbinder::unbind);
+        super.onDestroyView();
+    }
+
+    @Override
+    public void inject() {
+        component().inject(this);
+    }
+
+    @NonNull
+    @Override
+    protected HomeFragmentComponent createComponent() {
+        return DaggerHomeFragmentComponent.builder()
+                                          .homeActivityComponent(
+                                                  ((HomeActivity) getActivity()).component())
+                                          .baseFragmentModule(new BaseFragmentModule(this))
+                                          .build();
+    }
+
+    @NonNull
+    @Override
+    protected HomeFragmentViewModel viewModel() {
+        return get(homeFragmentViewModel);
+    }
+
+    @NonNull
+    @Override
+    protected Binder binder() {
```

Comment:
`Binder` is so overloaded, we should consider renaming that class. It could be anything from an Android Service `Binder` or a ButterKnife related entity.